.github/workflows/cifuzz.yml

```diff
@@ -47,16 +50,25 @@ jobs:
           keep-unaffected-fuzz-targets: true
           sanitizer: ${{ matrix.sanitizer }}
           architecture: ${{ matrix.architecture }}
+          output-sarif: true
       - name: Run Fuzzers (${{ matrix.sanitizer }})
         uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
         with:
           oss-fuzz-project-name: 'systemd'
           fuzz-seconds: 600
           dry-run: false
           sanitizer: ${{ matrix.sanitizer }}
+          output-sarif: true
       - name: Upload Crash
         uses: actions/upload-artifact@83fd05a356d7e2593de66fc9913b3002723633cb
         if: failure() && steps.build.outcome == 'success'
         with:
           name: ${{ matrix.sanitizer }}-${{ matrix.architecture }}-artifacts
           path: ./out/artifacts
+      - name: Upload Sarif
+        if: always() && steps.build.outcome == 'success'
+        uses: github/codeql-action/upload-sarif@9a866ed4524fc3422c3af1e446dab8efa3503411
```

Comment:
@jonathanmetzman could you pin it to 0225834cc549ee0ca93cb085b92954821a145866 to make it consistent with the other workflows where `upload-sarif` is used? (Dependabot tends to be confused by the same actions in different workflows pointing to different commits). It got bumped to that commit in https://github.com/systemd/systemd/pull/27879/files#diff-649a84adf982ddf68273f1bb0de882a20e97b06632b5432a6a1504a3f23cb2e6 as far as I can see.

Could you also squash the commits?

Other than that the PR looks good to me. If it looks good to @jamacku as well I think it should be good to go.

Reply:
> Other than that the PR looks good to me

On second thought I think it would maybe be better to bring backtraces first. Without them alerts are kind of cryptic. Dunno. I'm on the fence.